examples/multisig/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "local-project-test",
+  "version": "0.0.1",
+  "main": "index.js",
+  "license": "MIT",
+  "devDependencies": {
+    "chai": "^4.2.0"
```

Comment:
this is not used. we can remove`devDependencies`

Reply:
ok